### PR TITLE
add manifest to ensure pre-trained networks are available

### PR DESCRIPTION
**Pull request by oliche**
This allows pip users to re-use the pretrained denoisers

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
 click
+dredge
```
Comment (cwindolf) [inline requirements.txt:3]: I'm a bit confused here -- we don't have dredge on pypi

File: src/spike_psvae/chunk_features.py [hunk 2]
```diff
@@ -76,7 +76,7 @@ def handle_which_wfs(self, subtracted_wfs, cleaned_wfs, denoised_wfs):
                 f"which_waveforms={self.which_waveforms} not in ('subtracted', 'cleaned', denoised)"
             )
         if not self.tensor_ok and torch.is_tensor(wfs):
-            wfs = wfs.cpu().numpy()
+            wfs = wfs.cpu().detach().numpy()
```
Comment (cwindolf) [inline src/spike_psvae/chunk_features.py:79]: you may want to make sure that you have requires_grad=False on your waveforms and model (.eval()). (you can also call torch.set_grad_enabled(False)) --- detach is only necessary when gradients are being used

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1 @@
+graft src/dartsort/pretrained
```
Comment (cwindolf) [inline MANIFEST.in:1]: thanks for this. lead me here: https://stackoverflow.com/a/14159430/3979938 . who knew! i had the package-data part in my pyproject.toml but not this thing

**Comment by cwindolf** [comment]
I would only add that spike-psvae is not actively maintained and I'd suggest that we move your workflow to something dartsort-based, but I'm happy to merge this? thanks for the MANIFEST.in thing!

File: requirements.txt (hunk 1 above)
Comment (oliche) [reply requirements.txt:3]: Oups: 

https://pypi.org/project/dredge/

We had the same issue with one of our modules !

**Comment by oliche** [comment]
Yes I agree with moving on with the dartsort implementation. Let's close this PR and leave this branch in oblivion !
